lib/tasks/prune_reports.rake

```diff
@@ -43,6 +64,26 @@ UNITS:
       errors << "You must specify the unit of time, .e.g.: unit={#{known_units}}" \
     end
 
+    if ( ENV['status'] && ! ENV['condition'] ) || ( ENV['condition'] && ! ENV['status'] )
+      errors << "You must specify status AND condition!"\
```

Comment:
The trailing \ isn't needed. (I see it above in the time units line, that looks like a mistake.)